Source/Scene/ModelExperimental/ModelExperimentalNode.js

```diff
@@ -180,6 +183,17 @@ Object.defineProperties(ModelExperimentalNode.prototype, {
     },
   },
 
+  /**
+   * A transform from the node's local space to the model's scene graph space.
+   * This is the product of transformToRoot * transform
+   * @readonly
```

Comment:
Missing `@memberof ModelExperimentalNode.prototype` and `@type {Matrix4}` here.

This is also nitpicky, but can we put a period after the second sentence for consistency? :')